### PR TITLE
Remove non-frac grid coupled tests

### LINKED ISSUE #581: convert 4 existing cpld_control tests to fractional grid; drop existing non-frac RT tests for S2S, S2SW
## Description

The fractional grid implementation for S2S,S2SW is the default going forward. It has been well-tested in ufs-weather since January, 2021. We should drop any non-fractional grid coupled tests. 

## Solution

Convert the ``cpld_2threads``, ``cpld_decomp``, ``cpld_satmedmf`` and ``cpld_ca`` tests to use the fractional grid. This will change the baselines for these tests. 

Drop the following tests:

```
cpld_control
cpld_restart
cpld_control_c192
cpld_restart_c192
cpld_control_c384
cpld_restart_c384
cpld_bmark
cpld_restart_bmark
cpld_bmark_wave
cpld_debug
```

### PR DESCRIPTION
**Pull request by DeniseWorthen**
# PR Checklist

- [x] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [x] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [x] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [ ] If new or updated input data is required by this PR, it is clearly stated in the text of the PR.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

- makes ``nems_frac`` the default CPLMODE and sets ``FRAC_GRID=true`` as the default
- removes non-frac grid coupled regression tests; drops identifier "frac" from remaining test names
- reduces wall clock for cpld_debugfrac. This test runs in about 5 mins but wall clock was set at 60mins.
- reduce cpld_control_c384 to 12 hour forecast
- removes non-v16 benchmark and non-frac 35d tests 
- comments out cpld_2threads test (Issue #622)
- adds ``ktherm`` and ``tfrz_option`` for setting up mushy thermodynamics in CICE ``ice_in``. The defaults are set to current values ``ktherm=1`` and ``tfrz_option=linear_salt``.

### Issue(s) addressed

- fixes #581 

## Testing

Pre-commit testing for gaea.intel shows that the following tests do not reproduce:

```
cpld_ca
cpld_satmedmf
cpld_decomp
cpld_2threads
cpld_control_wave
```

This is the expected result since these are the low-resolution tests which are currently running with non-frac grid. All other coupled tests pass the current develop branch comparison (develop-20210513).

A test baseline was created for the new suite of coupled tests for gaea.intel and all test passed against the test baseline.

- [ ] hera.intel
- [ ] hera.gnu
- [ ] orion.intel
- [ ] cheyenne.intel 
- [ ] cheyenne.gnu
- [ ] gaea.intel 
- [ ] jet.intel
- [ ] wcoss_cray
- [ ] wcoss_dell_p3

## Dependencies

If testing this branch requires non-default branches in other repositories, list them. Those branches should have matching names (ideally).

Do PRs in upstream repositories need to be merged first?

- None

### CODE REVIEW

File: tests/rt.conf [hunk 1]
```diff
@@ -3,46 +3,35 @@
...
-RUN     | cpld_bmarkfrac_v16_nsst                                                                                                 | - wcoss_cray                            | fv3 |
 RUN     | cpld_restart_bmarkfrac_v16                                                                                              | - wcoss_cray                            |     | cpld_bmarkfrac_v16
+RUN     | cpld_bmarkfrac_v16_nsst                                                                                                 | - wcoss_cray                            | fv3 |
 
 COMPILE | APP=S2SW SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled,FV3_GFS_v16_coupled            | - wcoss_cray                            | fv3 |
-RUN     | cpld_bmark_wave                                                                                                         | - wcoss_cray                            | fv3 |
 RUN     | cpld_bmarkfrac_wave                                                                                                     | - wcoss_cray                            | fv3 |
```
Comment (JessicaMeixner-NOAA) [inline tests/rt.conf]: Do we still need the non-v16 configurations of the benchmarks?

Comment (DeniseWorthen) [reply tests/rt.conf]: That got removed in a second round of cleaning up. I'm not sure why the PR still shows it there.

**Comment by JessicaMeixner-NOAA** [comment]
@DeniseWorthen my only comment would be if we want to go ahead and also remove the non-v16 set-ups where we already have a v16 corresponding test so: 
cpld_bmarkfrac_wave  
cpld_bmarkfrac 
cpld_restart_bmarkfrac 
and then, maybe even: 
cpld_controlfrac_c384 
cpld_restartfrac_c384 
which is maybe redundant from cpld_bmarkfrac_v16 tests as well?   I think the difference between all of these (besides waves, restart) is just different atm physics options, but I could be wrong and maybe  this is all out of scope of this PR and planned for a later date?    

All the changes as is look good to me, however I did not "approve" as the full regression tests have not been run.

**Comment by DeniseWorthen** [comment]
I think some of these are ripe for removal, yes. I was leaving some until we finish w/ Jun's global RT test update (PR #561).

**Comment by junwang-noaa** [comment]
I'd suggest: 1) keep one non-frac test as the current GFSv16 test is still
non-frac. It might be easy if we'd like to checking something with
standalone atm run. 2) let's start the second round coupled test clean up
after #561 commit to use the GFSv16 opn namelist.

**Comment by DeniseWorthen** [comment]
For checking against a standalone run, it is easy to turn a frac grid sandbox into a non-frac grid sandbox by switching the two parameters (coupling_mode in nems.configure and frac_grid in input.nml). Would that be sufficient?

**Comment by DeniseWorthen** [comment]
Closed. Replacement PR #623 without wave update

**Pull request closed without merging**